src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class WorkWithFile {
+    private static final String SUPPLY_WORD = "supply,";
+    private static final String BUY_WORD = "buy,";
+    private static final String RESULT_WORD = "result,";
+    private String result = "";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        ArrayList<String> list = new ArrayList<>();
+        File myFile = new File(fromFileName);
+        Scanner scanner;
+        try (FileReader filereader = new FileReader(myFile)) {
+            scanner = new Scanner(filereader);
+            while (scanner.hasNextLine()) {
+                list.add(Arrays.toString(scanner.nextLine().split(System.lineSeparator())));
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file: " + fromFileName, e);
+        }
+
+        String[] words = new String[list.size()];
+        words = list.toArray(words);
```

Comment:
It's redundant to make an array from your list, cause when you read the file, you are creating a list of strings. You can easy use this List to create a result